Sources/CoreMetrics/Metrics.swift

```diff
@@ -302,6 +318,13 @@ public extension Timer {
         let handler = MetricsSystem.factory.makeTimer(label: label, dimensions: dimensions)
         self.init(label: label, dimensions: dimensions, handler: handler)
     }
+
+    /// Signal the underlying metrics library that this recorder will never be updated again.
```

Comment:
```suggestion
    /// Signal the underlying metrics library that this timer will never be updated again.
```

Reply:
(just trying out this cool GitHub 'suggestion' feature by just plagiarising Ian's suggestion :P )